frontends/election-manager/src/components/election_manager.tsx

```diff
@@ -123,6 +111,23 @@ export function ElectionManager(): JSX.Element {
       );
     }
 
+    if (!election || !configuredAt) {
+      return (
+        <Switch>
+          <Route exact path={routerPaths.electionDefinition}>
+            <UnconfiguredScreen />
+          </Route>
+          <Route exact path={routerPaths.settings}>
+            <SettingsScreen />
+          </Route>
+          <Route exact path={routerPaths.logs}>
+            <LogsScreen />
+          </Route>
+          <Redirect to={routerPaths.electionDefinition} />
+        </Switch>
+      );
+    }
+
```

Comment:
There's one branch that I know I haven't yet accounted for: a (non-super) admin auths with their card when there's no election definition. This will get handled as part of https://github.com/votingworks/vxsuite/issues/1928